src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/scatter_nd_update.cpp

```diff
@@ -179,12 +220,51 @@ const std::vector<ElementType> constantPrecisions = {
     ElementType::i32,
 };
 
-INSTANTIATE_TEST_SUITE_P(smoke_scatterndupdate_CompareWithRefs_dynamic, ScatterNDUpdateLayerGPUTest,
+const std::vector<ScatterUpdateLayerParams> scatterEmptyInput1_2Params = {
+    ScatterUpdateLayerParams{
+        ScatterUpdateShapes{
+            {{-1, -1, -1, -1}, {{ 100, 256, 14, 14 }}},
+            {{-1, 1}, {{ 0, 1 }}},
+            {{-1, 256, 14, 14}, {{ 0, 256, 14, 14 }}}
+        },
+        IndicesValues{ 0 },
+        Scatterupdate_type::ND
+    },
+#if 0 // ScatterELementsUpdate doesn't support dynamic shape yet. Need to enable when it supports.
+    ScatterUpdateLayerParams{
+        ScatterUpdateShapes{
+            {{-1, -1, -1, -1}, {{ 100, 256, 14, 14 }}},
+            {{-1, -1, 14, 14}, {{ 0, 256, 14, 14 }}},
+            {{-1, 256, 14, 14}, {{ 0, 256, 14, 14 }}},
+            {{1}, {{0}}}
+        },
+        IndicesValues{ 0 },
+        Scatterupdate_type::Elements
+    },
+#endif
```

Comment:
How about separate the TC and add prefix DISABLE instead of if 0?

Reply:
Updated. Thanks!